server/integration/jgroups/src/main/resources/subsystem-templates/cloud-jgroups.xml

```diff
@@ -104,7 +104,9 @@
                 <protocol type="FRAG3" />
             </stack>
             <stack name="kubernetes">
-                <?TCP-TRANSPORT?>
+                <transport type="TCP" socket-binding="jgroups-tcp" >
+                    <property name="logical_addr_cache_expiration">360000</property>
```

Comment:
I guess @tristantarrant might be worried about this one, but @belaban also changed the timeout in JGroups so it will be removed as soon as we upgrade to `4.0.2`.